modules/nw-metallb-operator-custom-resources.adoc

```diff
@@ -12,9 +16,32 @@ If the instance is deleted, the Operator removes MetalLB from the cluster.
 MetalLB requires one or more pools of IP addresses that it can assign to a service when you add a service of type `LoadBalancer`.
 When you add an `AddressPool` custom resource to the cluster, the MetalLB Operator configures MetalLB so that it can assign IP addresses from the pool.
 An address pool includes a list of IP addresses.
-The list can be a single IP address, a range specified in CIDR notation, a range specified as a starting and ending address separated by a hyphen, or a combination of the three.
+The list can be a single IP address that is set using a range, such as 1.1.1.1-1.1.1.1, a range specified in CIDR notation, a range specified as a starting and ending address separated by a hyphen, or a combination of the three.
```

Comment:
@sabinaaledort does this make the single IP address statement accurate?

Reply:
yes, sounds good to me

Reply:
Ok I'll also make a separate PR for this same fix in 4.9!